src/Fossa/API/Types.hs

```diff
@@ -151,7 +151,9 @@ instance ToJSON ArchiveComponents where
   toJSON ArchiveComponents{..} =
     object
       [ "archives" .= archiveComponentsArchives
-      , "fullFiles" .= archiveComponentsUpload
```

Comment:
Based on the previous code: https://github.com/fossas/fossa-cli/pull/1423/files#diff-78dfaf45d3d47ee1eb7bc8a678e57e40022f78c8c394da5c9adb61582e3c63c4L154

I've changed the json value for `archiveComponentsUpload` to be set to a boolean.